pyEpiabm/pyEpiabm/tests/test_utility/test_covidsim_kernel.py

```diff
@@ -0,0 +1,41 @@
+import unittest
+import numpy as np
+from parameterized import parameterized
+
+from pyEpiabm.utility.covidsim_kernel import SpatialKernel
+
+numReps = 5
+
+
+class TestSpatialKernel(unittest.TestCase):
+    """Test the 'SpatialKernel' class.
+    """
+    @parameterized.expand([(np.random.rand(1) * numReps,)
+                          for _ in range(numReps)])
+    def test_kernel(self, dist):
+        dist = 10 * dist[0]
+        dist_object = SpatialKernel()
+        value = dist_object.weighting(dist)
```

Comment:
Could just pass in dist[0] here - not sure the need for multiplying by 10, and given our default grid is 0 to 1 the `np.random.rand` is rather apt actually?